blog/arquillian-cube-1.2.0.textile

```diff
@@ -0,0 +1,32 @@
+---
+date: 2017-05-11
```

Comment:
I see your point. Let's see what else do we have in the line and maybe I will adjust the dates